### ISSUE TITLE
Haploid problem

### ISSUE THREAD

**Issue by cfz1998** [stschiff/msmc2#53]
Hi! @stschiff

Does MSMC2 work for haploid?  My organisms is haploid.

Thank you for your reply!
                        --CFZ

**Comment by stschiff** [comment]
Yes, if it is recombining. If not, then you don't need MSMC, you can just use any phylogenetic software or build a neighbour-joining tree.

**Comment by cfz1998** [comment]
Hi！@stschiff  
Should make a “pseudo-diploid”(merge two sample genotypes) after `bamCaller.py`? Or just simply double each sample genotype and pick only one haplotype to msmc2?
Like: `msmc2 -t 40 -p 1*2+25*1+1*2 -i 100 -I 0,2,4,6,8,10,12,14` 8 samples

--CFZ

**Comment by cfz1998** [comment]
And, if merged two Haploid samples to a “pseudo-diploid”. How to deal with the “mask.bed.gz” from `bamCaller.py`?
Simply merged(Interval merging) the “mask.bed.gz” from two Haploid samples?
